guides/tutorials/codelabs/halyard-getting-started/index.md

```diff
@@ -20,6 +20,11 @@ In our scenario, we want to create a Spinnaker instance and set it up as follows
 
 For this exercise we will be operating entirely within one GCP project, and use Google Container Engine (GKE) as our Kubernetes cluster. 
 
+{% include figure 
```

Comment:
I could - I wanted a caption though, and the `include figure` provides that with less fuss. Take a look at https://github.com/spinnaker/spinnaker.github.io/blob/master/_includes/figure